Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -861,13 +870,19 @@ private void RenderNodeChildren (HtmlTextWriter writer, WebTreeNode node, bool i
       else
         writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassNodeChildren);
       writer.AddAttribute(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.Group);
+      writer.AddAttribute(HtmlTextWriterAttribute.Id, CreateSubTreeNodeID(parentNodeID));
       writer.RenderBeginTag(HtmlTextWriterTag.Ul); // Begin child nodes
 
       RenderNodes(writer, node.Children, false, nodeIDAlgorithm, nestingDepth + 1);
 
       writer.RenderEndTag(); // End child nodes
     }
 
+    private string CreateSubTreeNodeID (string nodeID)
+    {
+      return $"Node_{nodeID}_SubTree";
```

Comment:
We shouldn't add anything to the ID in the front. IDs in ASP.NET are hierarchical and container all the IDs of parent elements from top to bottom. e.g. `Body_MyContainer_MyList_MyItem`

Reply:
There was nothing added to the front in comparison to before, do you mean to say that it should be `Node_SubTree_0000-.....-0000`?